version 4.2 structure (#1127)
neo4j/neo4j-documentation: cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

label:functionality[]
label:updated[]
[source, cypher, role="noheader"]
----
SHOW ROLE name PRIVILEGES
----
a|
Can now handle multiple roles.
[source, cypher, role="noheader"]
----
SHOW ROLES n1, n2, ... PRIVILEGES
----

a|
label:functionality[]
label:updated[]
[source, cypher, role="noheader"]
----
SHOW USER name PRIVILEGES
----
a|
Can now handle multiple users.
[source, cypher, role="noheader"]
----
SHOW USERS n1, n2, ... PRIVILEGES
----

a|
label:functionality[]
label:updated[]
[source, cypher, role="noheader"]
----
round(expression, precision)
----
a|
The `round()` function can now take an additional argument to specify rounding precision.

a|
label:functionality[]
label:updated[]
[source, cypher, role="noheader"]
----
round(expression, precision, mode)
----
a|
The `round()` function can now take two additional arguments to specify rounding precision and rounding mode.

a|
label:syntax[]
label:added[]
[source, cypher, role="noheader"]
----
DEFAULT GRAPH
----
a|
New optional part of the Cypher commands for <<administration-security-administration-database-privileges,database privileges>>.

a|
label:syntax[]
label:added[]
[source, cypher, role="noheader"]
----
0o...
----
a|

> Cypher will now interpret literals with prefix `0o` as an octal integer literal.

Reviewer: ```suggestion
Cypher now interprets literals with prefix `0o` as an octal integer literal.
```

a|
label:syntax[]
label:deprecated[]
[source, cypher, role="noheader"]
----
0...
----
a|
Replaced by `+0o...+` (see above).

a|
label:syntax[]
label:deprecated[]
[source, cypher, role="noheader"]
----
0X...
----
a|

> Only `+0x...+` (lowercase x) will be supported.

Reviewer: ```suggestion
Only `+0x...+` (lowercase x) is supported.
```

a|
label:syntax[]
label:added[]
[source, cypher, role="noheader"]
----
SET [PLAINTEXT \| ENCRYPTED] PASSWORD
----
a|

> For `CREATE USER` and `ALTER USER` it is now possible to set (or update) a password when the plaintext password is unknown, but the encrypted password is available.

Reviewer: ```suggestion
For `CREATE USER` and `ALTER USER`, it is now possible to set (or update) a password when the plaintext password is unknown, but the encrypted password is available.
```
